### ISSUE TITLE
Character Appearance Changed

### ISSUE THREAD

**Issue by absy2k15** [rockfireredmoon/iceee#162]
So I created a character with the ears and tail set to max. The next day, my character's ears and tails size changed to medium. How can I restore them back to max setting which is the one I liked.
![capture](https://cloud.githubusercontent.com/assets/16470444/12028656/b1c09768-ad8f-11e5-8201-11f2a0c339d9.JPG)

**Comment by Heathendel** [comment]
yes thats a bug we've been aware of for some time now.  the tail/ear size
feature is something we added to character creation that wasnt in the
original but unfortunately we ran into that odd bug where it resets we are
looking into however we've been more focused on game mechanics and further
expansions.  We do plan on heavily revamping character creation to make the
customization way better then it currently is and the bug will be fixed at
that time

**Comment by absy2k15** [comment]
I see. Thanks for the fast response.
